packages/js/src/Modules/Verto/webrtc/helpers.ts

```diff
@@ -368,6 +368,210 @@ const sdpBitrateHack = (
+      .split('/');
+    const name = info[0];
+    const version = parseInt(info[1], 10);
+
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: true,
+    };
+  }
+
+  if (
+    navigator.userAgent.match(/firefox/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi) &&
+    !navigator.userAgent.match(/edg/gim)
+  ) {
+    const info = navigator.userAgent
+      .match(/firefox\/[0-9]+\./gim)[0]
+      .split('/');
+
+    const name = info[0];
+    const version = parseInt(info[1], 10);
+
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: false,
+    };
+  }
+
+  if (
+    navigator.userAgent.match(/safari/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi) &&
+    !navigator.userAgent.match(/edg/gim)
+  ) {
+    const name = navigator.userAgent.match(/safari/gim)[0];
+    const fullVersion = navigator.userAgent
+      .match(/version\/[0-9]+\./gim)[0]
+      .split('/');
+    const version = parseInt(fullVersion[1], 10);
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: true,
+    };
+  }
+
+  if (
+    navigator.userAgent.match(/edg/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi)
+  ) {
+    const info = navigator.userAgent.match(/edg\/[0-9]+\./gim)[0].split('/');
+    const name = info[0];
+    const version = parseInt(info[1], 10);
+
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: true,
+    };
+  }
+  throw new Error(
+    'This browser does not support @telnyx/webrtc. \nPlease, use Chrome|Firefox|Safari.'
+  );
+}
+
+function getWebRTCInfo() {
+  try {
+    const { browserInfo, name, version, supportAudio, supportVideo } = getBrowserInfo();
+    const PC = window.RTCPeerConnection;
+    const sessionDescription = window.RTCSessionDescription;
+    const iceCandidate = window.RTCIceCandidate;
+    const mediaDevices = window.navigator && window.navigator.mediaDevices;
+    const getUserMediaMethod =
+      navigator.getUserMedia ||
+      navigator.webkitGetUserMedia ||
+      navigator.msGetUserMedia ||
+      navigator.mozGetUserMedia;
+
+    return {
+      browserInfo,
+      browserName: name,
+      browserVersion: version,
+      supportWebRTC:
+        !!PC &&
+        !!sessionDescription &&
+        !!iceCandidate &&
+        !!mediaDevices &&
+        !!getUserMediaMethod,
+      supportWebRTCAudio: supportAudio,
+      supportWebRTCVideo: supportVideo,
+      supportRTCPeerConnection: !!PC,
+      supportSessionDescription: !!sessionDescription,
+      supportIceCandidate: !!iceCandidate,
+      supportMediaDevices: !!mediaDevices,
+      supportGetUserMedia: !!getUserMedia,
+    };
+  } catch (error) {
+    console.error(error.message);
+    return error.message;
```

Comment:
I would remove the `console.error` here, if the expectation is for library developers to handle errors.
```suggestion
    return error.message;
```